AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsView.swift

```diff
@@ -24,7 +24,7 @@ struct CreateSessionDetailsView: View {
                             sessionNameField
                             sessionTagsField
                         }
-                        if sessionContext.sessionType == SessionType.fixed { fixedSessionDetails }
+                        if sessionContext.sessionType == SessionType.mobile { fixedSessionDetails }
```

Comment:
I think you added this for testing and forgot to change it back :D 
Please, change it - it's not working for fixed session right now 🙃

Reply:
yees. oh my :/